### PR TITLE
Remove G12 recovery code

### PR DESCRIPTION
**Pull request by alex9smith**
https://trello.com/c/MJExPLo2/846-3-remove-g12-recovery-code-and-pii

Remove the scripts we [wrote for G12 recovery](https://github.com/alphagov/digitalmarketplace-scripts/pulls?q=is%3Apr+is%3Aclosed+G12+recovery). Reverts:

#646
#641
#639
#638
#628
#616
#610

### CODE REVIEW

File: dmscripts/helpers/supplier_data_helpers.py [hunk 1]
```diff
@@ -392,12 +390,3 @@ def unsuspend_suspended_supplier_services(record, suspending_user, client, logge
             logger.info(f"[DRY RUN] Would unsuspend service {service_id} for supplier {supplier_id}")
         else:
             client.update_service_status(service_id, new_service_status, "Unsuspend services helper")
-
-
-def get_email_addresses_for_supplier(api_client: DataAPIClient, supplier_id: int) -> List[str]:
```
Comment (alex9smith) [inline dmscripts/helpers/supplier_data_helpers.py:397]: I could leave this here, since it's a function that we might want to use in other scripts

**Comment by bjgill** [comment]
I'll look through this in a moment. Is this everything? Searching for 'g12', it looks as if https://github.com/alphagov/digitalmarketplace-scripts/pull/637 and https://github.com/alphagov/digitalmarketplace-scripts/pull/640 are still there...

**Comment by alex9smith** [comment]
> I'll look through this in a moment. Is this everything? Searching for 'g12', it looks as if #637 and #640 are still there...

So they are... Not sure how I missed those

File: dmscripts/helpers/supplier_data_helpers.py (hunk 1 above)
Comment (bjgill) [reply dmscripts/helpers/supplier_data_helpers.py:397]: Arguably my fault for including generic improvements within G12 PRs :slightly_smiling_face:

Not difficult to re-instate if we discover we need/want this in future.

**Review by bjgill — APPROVED** [review]
:fire:

**Pull request merged**